### DISCUSSION TITLE
Help, my project can run normally, but it does not load JS

### DISCUSSION THREAD

**Discussion by Lc555200** [vikejs/vike#1639] [Discussion]
Hello, I'm sorry to bother you, but I have encountered some problems, and I do not know how to solve them.  I introduced vike into the original project and developed it according to Vike-with-Pinia and Vike-Vue-Router.  Now the project can run without throwing errors, but it lacks the js call.  There is no data presented on the page after the project is running, and there is no request to send out.  I think there must be something wrong with my configuration.   Could you please provide some guidance, thank you so much!
![app](https://github.com/vikejs/vike/assets/45931231/603a2d6f-c1c1-4124-ad7d-7f75ceaaf68f)
![beforerender](https://github.com/vikejs/vike/assets/45931231/f2bbbf3b-03ed-4036-b4d6-2831d9cd7358)
![config](https://github.com/vikejs/vike/assets/45931231/5010c7c9-c1db-4d03-b22b-f13afa60fc5c)
![dev](https://github.com/vikejs/vike/assets/45931231/e95ee4f1-c3c2-4de7-94e4-67df3fbc5dc5)
![nodata](https://github.com/vikejs/vike/assets/45931231/5831738d-0009-4a9a-bb38-cda2c9ff6d4b)
![page](https://github.com/vikejs/vike/assets/45931231/a680f1b3-0a5c-43e1-a758-99cc1b4e3474)
![pagelayout](https://github.com/vikejs/vike/assets/45931231/8cde7d70-ad55-4090-9c2e-3080d9034bd5)
![renderclient](https://github.com/vikejs/vike/assets/45931231/725ae11d-1adf-446a-bed1-4b6fc73d4bae)
![renderhtml](https://github.com/vikejs/vike/assets/45931231/f8eae03f-de70-4f62-bce7-15557a29baf5)
![router](https://github.com/vikejs/vike/assets/45931231/8ac40b0e-3307-4081-9981-e2901a945982)
![usepagecontext](https://github.com/vikejs/vike/assets/45931231/602d26bd-45df-4fe9-a0f0-c0063c6f02a3)

**Comment by Lc555200** [comment]
![package](https://github.com/vikejs/vike/assets/45931231/95c607b0-6b03-4db4-af90-971aad1968a4)

**Comment by brillout** [comment]
This doesn't seem related to Vike, but feel free to let me know if you think it does.

**Reply by Lc555200** [reply]
Thank you for your reply. I also need to ask you whether the part about vike written in my project is correct？

**Reply by Lc555200** [reply]
![image](https://github.com/vikejs/vike/assets/45931231/aa91e22a-cd70-4738-ad6c-cd3eb19602dc)

Is there no problem loading only these two JS? Looking forward to your reply！
